### DISCUSSION TITLE
How to search using image parameters?

### DISCUSSION THREAD

**Discussion by armondavani** [zanllp/infinite-image-browsing#752] [Q&A] [answered]
Hello. The github description mentions "...search capabilities using image parameters."

In the IIB tab, if you click on an image it lists the "Params" on the side. 

But when doing advanced search, I don't see a way to leverage these params in the search. For example "seed", or "mask blur".

Thank you!

**Comment by zanllp** [comment]
Refer to the following, or you can also use regular expressions for searching.
<img width="655" alt="image" src="https://github.com/user-attachments/assets/b8d8234e-36a9-481a-883f-b7cfbd491682">

**Comment by armondavani** [comment]
@zanllp 

So are you not able to search by things like: 
seed
steps
cfgScale
Schedule type
etc

-even though these are documented under "params" for each image? I don't see these in the list of parameters in the extension above, and when I try to type and search by them, they don't appear in the autocomplete either.

Thank you!

**Comment by zanllp** [comment, marked as answer]
IIB has Fuzzy Search as a supplementary feature for Tag Search, allowing you to use regular expressions in Fuzzy Search to perform these queries.

![image](https://github.com/user-attachments/assets/d1c63483-3491-499c-8e28-5619cf6ea4a9)

**Comment by armondavani** [comment]
I see! Thank you! ❤️ 

Of note to anyone else who sees this, the space is important. 

`steps:20` won't work but `steps: 20` will.
